morph_tool/utils.py

```diff
@@ -111,7 +111,7 @@ def get_incomplete_sections(asc_morph_file, tag='Incomplete'):
             if tag in line:
                 for parsed_line in parsed_lines[::-1]:
                     match = re.match(
-                        r'[\s]*\(\s*(-?\d+.?\d*)\s+(-?\d+.?\d*)\s+(-?\d+.?\d*)\s+(-?\d+.?\d*)\s*\)',
+                        r'^\s*\(\s*(-?\d+.?\d*)\s+(-?\d+.?\d*)\s+(-?\d+.?\d*)\s+(-?\d+.?\d*)\s*\)',
```

Comment:
I would highly recommend using the `VERBOSE` compilation flag, and commenting the re:
https://docs.python.org/3/library/re.html#re.VERBOSE

Reply:
Thank you! I completely forgot about it.